Attendee Procedure for incident handling.md

```diff
@@ -0,0 +1,37 @@
+This procedure has been adopted from the Ada Initiative's guide titled "[Conference anti-harassment/Responding to Reports](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports)”.
+
+1\. Keep in mind that all conference staff will be wearing a conference t-shirt/button with the word “STAFF”
+on it (or otherwise clearly marked as staff). The staff will also be prepared to handle the incident.
+All of our staff are informed of the [code of conduct policy](code_of_conduct.md) and guide
+for handling harassment at the conference. *There will be a mandatory staff meeting onsite at the conference
+when this will be reiterated.*
+
+2\. Report the harassment incident (preferably in writing) to a conference staff member. All reports
+are confidential. Please do not disclose public information about the incident until the staff have
+had sufficient time in which to address the situation. This is as much for your safety and protection
+as it is the other attendees.
+
+When reporting the event to staff, try to gather as much information as available but do not
+interview people about the incident. Staff will assist you in writing the report/collecting information.
+
+The important information consists of:
+
+- Identifying information (name/badge number) of the participant doing the harassing
```

Comment:
Are your badges numbered?

Reply:
:thinking: No, but it may be a good idea, I'll leave this in.